### ISSUE TITLE
Damage textures displayed at 0,0,0

### ISSUE THREAD

**Issue by Goober5000** [scp-fs2open/fs2open.github.com#497]
The best way to describe this is with a picture:

![damage-at-origin](https://cloud.githubusercontent.com/assets/1878523/11760528/32e2a6b8-a06d-11e5-9034-fff4705e107c.png)

In short, placing the GTCa Warlock in FRED will cause three circles with the Damage texture to be displayed at the origin coordinates, regardless of where the ship itself is located.

At first I thought this was a problem with the model, but then I noticed the problem doesn't appear in 3.7.2.  Applying `git bisect` tracked it down to c620372357afd79ac988b60112d40b99d7efed32, which is Swifty's giant graphics commit from May.

Here's the GTCa Warlock modpack:
http://sa.freespacemods.net/warlock.zip

**Comment by SamuelCho** [comment]
Can anyone else reproduce this issue? I'm not seeing it on my end using this model.

**Comment by Goober5000** [comment]
Note that Damage.dds is in MVPs 2014.  Make sure those are on your path.

(Yarn and Woomeister didn't see the problem either, unfortunately.)

**Comment by MageKing17** [comment]
Maybe it's being caused by a specific command-line flag?

**Comment by Goober5000** [comment]
No, it still happens even when `-mod scroll` is the only flag.  Here are logs with and without MediaVPs.
http://staff.hard-light.net/goober5000/temp/fred2_open-mvps.zip
http://staff.hard-light.net/goober5000/temp/fred2_open-nomvps.zip

**Comment by Goober5000** [comment]
Heh, this is interesting.  While doing something unrelated, I loaded up FRED with no mods, then placed a retail Orion.  This is what happened:

![orion-cartoon](https://cloud.githubusercontent.com/assets/1878523/11772195/db35f0c4-a1e7-11e5-8c53-e6c07aa3b258.jpg)

**Comment by chief1983** [comment]
@SamuelCho any chance you were able to reproduce Goober's find?

**Comment by Goober5000** [comment]
Also, is there any way to break up c620372 into smaller sections for further bisection?

**Comment by Goober5000** [comment]
Hrm.  I had hoped that #546 might fix this, but it shows the same behavior as before.  We'll need @SamuelCho  to take a look as soon as he gets a chance.

**Comment by chief1983** [comment]
I couldn't reproduce with the Orion last night, not sure I was attempting correctly.  Fired up FRED 3.7.4 RC1 with no arguments, moved the Ulysses out of the way, and dropped an orion off center.  Had to zoom out and move it more away since it was covering the origin but I never saw any texture rendering issues no matter what I tried to do, selected or deselected.

**Comment by Goober5000** [comment]
It must have been fixed at some point.  I checked the latest FRED and the problem no longer appears.

**Comment by Goober5000** [comment]
It must have been fixed at some point.  I checked the latest FRED and the problem no longer appears.
